backend/tests/integration/hal/conftest.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2018 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+
+import json
+import os
+
+import pkg_resources
+import pytest
+from invenio_db import db
+from invenio_pidstore.models import PersistentIdentifier
+from invenio_records.models import RecordMetadata
+
+IS_VCR_ENABLED = True
+IS_VCR_EPISODE_OR_ERROR = False  # False to record new cassettes.
+
+
+@pytest.fixture(scope="session")
+def vcr_config():
+    if IS_VCR_EPISODE_OR_ERROR:
+        record_mode = "none"
+    else:
+        record_mode = "new_episodes"
+
+    if not IS_VCR_ENABLED:
+        # Trick to disable VCR.
+        return {"before_record": lambda *args, **kwargs: None}
+
+    return {
+        "decode_compressed_response": True,
+        "filter_headers": ("Authorization", "User-Agent"),
+        "ignore_hosts": (
+            "localhost",
+            "elasticsearch",
+            "postgres",
+            "redis",
+            "indexer",
+            "cache",
+            "db",
+            "mq",
+            "hep-worker",
+            "hep-web",
+            "ui",
+            "next-web",
+            "next-worker",
+            "flower",
+        ),
+        "record_mode": record_mode,
+    }
```

Comment:
@oguzdemirbasci we have already [configuration](https://github.com/inspirehep/inspirehep/blob/master/backend/tests/conftest.py#L12) for that you don't need it

Reply:
neither the other two vcr config